_lessons/who-is-analytics-engineer.html

```diff
@@ -27,17 +27,24 @@
 ]
 ]
 
+???
+Slide setup: When did people first start hearing the term 'analytics engineering'? Analytics Engineering is relatively new. The role of Analytics Engineer is relatively new. To understand it, we need to understand how data teams traditionally looked. 
+
+Note: Lots of custom engineering work. Distance between business stakeholders and those doing the actual transformation of their data.
+
 ---
 # Modern data stack
 
 <img src="/ui/img{{page.id}}/premodern-data-stack.svg" class="img-center">
 
 ???
-- MPP warehouses replaced the need for engineers to maintain warehouse infrastructures
-- Stitch and Fivetran got data into your warehouse, but the data was then in a source conformed format
-- BI tools (Looker) allowed consumers to do their own deep dives into data
+- Reliable out of the box data loader tools like Stitch and Fivetran made it easy to get data into your warehouse, but the data is in a source conformed format.
+- MPP (massively parallel processing) warehouses such as Snowlfake, Redshift, and BigQuery replaced the need for engineers to maintain warehouse infrastructures.
```

Comment:
```suggestion
- MPP (massively parallel processing) warehouses such as Snowflake, Redshift, and BigQuery replaced the need for engineers to maintain warehouse infrastructures.
```
quick-fix: small typo